OCCommunicationLib/OCCommunicationLib/OCWebDavClient/OCWebDAVClient.h

```diff
@@ -48,7 +48,7 @@ extern NSString *OCWebDAVCreationDateKey;
 /** The key for last modification date of an entity. */
 extern NSString *OCWebDAVModificationDateKey;
```

Comment:
Many warnings in this class for example in the line 49 class with the message "Pointer is missing a nullability type specifier (_Nonull, _Nullable, or _Null_specified)". Please, consider adding _Nullable in the methods